.github/workflows/ci.yml

```diff
@@ -107,3 +107,11 @@ jobs:
         run: cargo semver-checks check-release -p bootloader_api
       - name: Check semver
         run: cargo semver-checks check-release
+
+  typos:
+    name: Check typos
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: cargo install typos-cli --locked
+      - run: typos
```

Comment:
Any reason to not use the [GitHub action](https://github.com/crate-ci/typos/blob/master/docs/github-action.md) provided by that crate? E.g.:

```yaml
- name: Check spelling
  uses: crate-ci/typos@v1.0.4
```

I'm fine with the `cargo install` approach too if it works better.

Reply:
I didn't think that, thanks! I will publish a new commit.

Reply:
I created the commit

Reply:
The spelling test passes!

Reply:
Great, thanks! Looks like it's quicker this way.